sources/packages/web/src/components/generic/formio.vue

```diff
@@ -20,6 +20,9 @@ export default {
     data: {
       type: Object,
     },
+    readOnly: {
+      type: String,
+    },
```

Comment:
did you check, what happens when `readOnly is ""` (when `readOnly` is not passed)?